packages/common/src/editors/dateEditor.ts

```diff
@@ -201,7 +201,8 @@ export class DateEditor implements Editor {
         });
       }
 
-      setTimeout(() => {
+      // INFO: Fixes issue no 2
+      Promise.resolve().then(() => {
```

Comment:
oh that's cool, does that do 1 CPU cycle? Because that's typically why I use `setTimeout`

Reply:
the js event loop consists of cyclic tasks. Every such task is performing one render action. If you say setTimeout(..., 100) it skips 100ms and puts your task onto the macrotask queue.

There is a priority lane though, called microtask queue, which is a happening within every macrotask. 
So all promises are pushed towards the end of a macrotask and executed all together before the next macrotask starts. Even nested promises.

Hence why often an optimization when working with DOM Elements is to stuff things into microtasks and only render them once alltogether in the next macrotask

Reply:
BTW, after reading the article it seems that we could instead use `queueMicrotask(f)` which seems shorter, isn't it better?

Reply:
its the same but yeah more expressive 👍